src/views/CountryAdditionalData/index.tsx

```diff
@@ -1,12 +1,157 @@
+{
+    type Props as LinkProps,
+} from '#components/Link';
+import {
+    createStringColumn,
+} from '#components/Table/ColumnShortcuts';
+import HtmlOutput,
+{
+    type Props as HtmlOutputProps,
+} from '#components/HtmlOutput';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type CountrySnippetType = NonNullable<GoApiResponse<'/api/v2/country_snippet/'>['results']>[number];
+type CountryContactsType = NonNullable<CountryResponse['contacts']>[number];
+type CountryLinksType = NonNullable<CountryResponse['links']>[number];
 
 // eslint-disable-next-line import/prefer-default-export
 export function Component() {
+    const { countryId } = useParams<{ countryId: string }>();
+    const { countryResponse } = useOutletContext<CountryOutletContext>();
+
+    const {
+        pending: countrySnippetPending,
+        response: countrySnippetResponse,
+    } = useRequest({
+        skip: isNotDefined(countryId),
+        url: '/api/v2/country_snippet/',
+        query: { country: Number(countryId) } as never,
+    });
+
+    const hasCountrySnippet = (countrySnippetResponse?.results
+        && (countrySnippetResponse.results.length > 0) && !countrySnippetPending);
+
+    const hasCountryContacts = (countryResponse?.contacts
+        && (countryResponse.contacts.length > 0));
+
+    const hasCountryLinks = (countryResponse?.links
+        && (countryResponse.links.length > 0));
+
+    const strings = useTranslation(i18n);
+
+    const countrySnippetRendererParams = useCallback(
+        (_: number, data: CountrySnippetType): HtmlOutputProps => ({
+            className: styles.countrySnippetContent,
+            value: data.snippet,
+        }),
+        [],
+    );
+
+    const countryLinkRendererParams = useCallback(
+        (_: number, data: CountryLinksType): LinkProps => ({
+            to: data.url,
+            withExternalLinkIcon: true,
+            children: data.title,
+        }),
+        [],
+    );
+
+    const contactTableColumns = useMemo(
+        () => ([
+            createStringColumn<CountryContactsType, number>(
+                'name',
+                strings.columnName,
+                (item) => item.name,
+            ),
+            createStringColumn<CountryContactsType, number>(
+                'title',
+                strings.columnTitle,
+                (item) => item.title,
+            ),
+            createStringColumn<CountryContactsType, number>(
+                'ctype',
+                strings.columnType,
+                (item) => item.ctype,
+            ),
+            createStringColumn<CountryContactsType, number>(
+                'email',
+                strings.columnEmail,
+                (item) => item.email,
+            ),
+        ]),
+        [strings],
+    );
+
     return (
-        <div>
-            <UnderConstructionMessage
-                title="Country Additional Data"
-            />
+        <div className={styles.additionalContent}>
```

Comment:
className should match component name
```suggestion
        <div className={styles.countryAdditionalData}>
```